src/Our.Umbraco.Vorto/Web/UI/App_Plugins/Vorto/js/vorto.js

```diff
@@ -192,6 +202,10 @@
             }
         });
 
+        $scope.$watch("model.value.values", function () {
```

Comment:
The reason for watching is that this way the thick would show as soon as you change language.
Otherwise if you have a lot of languages you will only know which have content and which not only when you submit, which defeats a bit the purpose of the tickmarl